cvm/quantization/tfm_ops.py

```diff
@@ -89,6 +89,14 @@ def calculate_ops(self, op, **kwargs):
             op = Relu().calculate_ops(op, **kwargs)
         return op
 
+    def compile(self, op, **kwargs):
+        childs = kwargs['childs']
+        attrs = kwargs['attr']
+        act_type = attrs['act_type']
+        if act_type in ['relu']:
+            op_name, new_attrs = act_type, {}
```

Comment:
The Activation is a proxy op, please use Relu().compile function instead as the definition of rewrite.